src/main/java/baseball/repository/GameSetRespository.java

```diff
@@ -0,0 +1,52 @@
+package baseball.repository;
+
+import static baseball.message.Message.BEGIN_RANGE;
+import static baseball.message.Message.CAPACITY_STR;
+import static baseball.message.Message.END_RANGE;
+import static baseball.message.Message.GAME_STATUS_STR;
+
+import java.util.HashMap;
+
+public class GameSetRespository {
+
+    // 게임 관련 환경 설정
+    private static final HashMap<String, Integer> gameSetStore = new HashMap<>();
+
+    // 게임 상태
+    private static final HashMap<String, Boolean> gameStatus = new HashMap<>();
+
+    // 해당 리포지토리는 최초 생성 될 때만 초기화
+    static {
+        saveGameSet(CAPACITY_STR, 3);
+        saveGameSet(BEGIN_RANGE, 1);
+        saveGameSet(END_RANGE, 9);
+    }
```

Comment:
이렇게 생성하는 동시에 값을 초기화 하는 방법도 있어요 :)
```suggestion
    // 해당 리포지토리는 최초 생성 될 때만 초기화
    private static final HashMap<String, Boolean> gameStatus = new HashMap<>(){{
        gameStatus.put(CAPACITY_STR, 3);
        gameStatus.put(BEGIN_RANGE, 1);
        gameStatus.put(END_RANGE, 9);
    }}
```

Reply:
생성할 때 바로 넣어주는 방식이 있군요.
잘 배워 갑니다.